source/interact-data/specify-query.txt

```diff
@@ -0,0 +1,476 @@
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-symbol-query
+   :end-before: end-symbol-query
+   :language: ruby
+   :dedent:
+
+Fields
+------
+
+This section describes how to perform queries on fields with different
+types of values.
+
+Defined Fields
+~~~~~~~~~~~~~~
+
+.. TODO add link to Fields page
+
+To query on a field, the field does not need to be in the
+the model class definition. However, if a field is defined in
+the model class, {+odm+} coerces query values to match the defined field
+types when constructing the query.
+
+The following code specifies a string value when querying on the
+``founded`` field. Because the ``founded`` field is defined in the model
+class to have ``Integer`` values, {+odm+} coerces the string ``'2020'``
+to ``2020`` when performing the query:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-defined-field-query
+   :end-before: end-defined-field-query
+   :language: ruby
+   :dedent:
+
+Raw Values
+~~~~~~~~~~
+
+To bypass {+odm+}'s query type coercion behavior and query
+directly for the raw-typed value in the database, wrap the query value in
+the ``Mongoid::RawValue`` class, as shown in the following code:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-raw-field-query
+   :end-before: end-raw-field-query
+   :language: ruby
+   :dedent:
+
+This behavior can be useful when working with legacy data.
+
+.. _mongoid-query-field-alias:
+
+Field Aliases
+~~~~~~~~~~~~~
+
+.. TODO update links
+
+Queries follow the :ref:`storage field names <storage-field-names>`
+and :ref:`field aliases <field-aliases>` that you might have set in your
+model class definition.
+
+The ``id`` and ``_id`` fields are aliases, so you can use either field
+name in queries:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-id-field-query
+   :end-before: end-id-field-query
+   :language: ruby
+   :dedent:
+
+Embedded Documents
+~~~~~~~~~~~~~~~~~~
+
+To query on values of fields of embedded documents, you can use dot
+notation. The following code retrieves documents in which the ``name``
+field of the embedded ``Manager`` document is ``'Smith'``:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-embedded-query
+   :end-before: end-embedded-query
+   :language: ruby
+   :dedent:
+
+The following code demonstrates how to use a symbol operator when
+querying on an embedded field:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-embedded-ne-query
+   :end-before: end-embedded-ne-query
+   :language: ruby
+   :dedent:
+
+.. note::
+
+   Queries always return top-level model instances, even if all the
+   conditions reference embedded document fields.
+
+.. _mongoid-query-logical-operations:
+
+Logical Operations
+------------------
+
+{+odm+} supports ``and()``, ``or()``, ``nor()``, and ``not()`` logical
```

Comment:
[Style guide](https://www.mongodb.com/docs/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#create-short-paragraphs) - split this into a list since it's more than three objects